Core/NoOpRunner.Core/Enums/Entity.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace NoOpRunner.Core.Enums
 {
-    public enum Entity
+    public enum Entity//meh?
```

Comment:
Elaborate? xD

Reply:
MUDA

Reply:
Useless now, maybe use for UI type?

Reply:
You could probably safely remove anything unused, it's just an enum, if we need it we can reintroduce it